R/plots.R

```diff
@@ -270,19 +273,22 @@ plotLoadResidualsCM <- function(...){
 #'   
 #' @param load.model use this to pull names of date column and constituent name
 #' @param observations these are the observations to plot the residuals for
+#' @param residuals the observation residuals 
 #' @param dateField name of the date field in observations
 #' @param verbose debugging flag
 #' @param day.of.year flag if true will plot residuals by the day of year to,
 #'   can help user look for seasonality
-plotResidualsCM <- function(type, load.model, observations, dateField="Date", verbose=FALSE,  xObservations=FALSE, xFlow=FALSE, day.of.year=FALSE){
- 
+plotResidualsCM <- function(type, load.model, observations,resids, dateField="Date", verbose=FALSE,  xObservations=FALSE, xFlow=FALSE, day.of.year=FALSE){
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes_string=geom_point=yday=xlab=NULL
   dateName <- load.model$dates
   soluteName <- load.model$constituent
   flowName <- load.model$flow
   date_col <- names(which(sapply(c(dateField,  "Date",  dateName,"Period"), function(field) { !is.null(observations[[field]]) } ))[1])
   estName <- ""
   if(type=="conc"){
-    resids <- predictSoluteFromRegression("conc", load.model, observations)
+    #resids <- predictSoluteFromRegression("conc", load.model, observations)
```

Comment:
this looks problematic to me. not a big deal, since i'll be rewriting this file in a week or two, but any idea why the line was in there in the first place?